examples/async-universal/webpack.config.js

```diff
@@ -0,0 +1,49 @@
+var path = require('path')
+var webpack = require('webpack')
+
+module.exports = {
+  devtool: 'cheap-module-eval-source-map',
+  entry: [
+    'webpack-hot-middleware/client',
+    './client/index.js'
+  ],
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: 'bundle.js',
+    publicPath: '/static/'
+  },
+  plugins: [
+    new webpack.optimize.OccurenceOrderPlugin(),
+    new webpack.HotModuleReplacementPlugin(),
+    new webpack.NoErrorsPlugin()
+  ],
+  module: {
+    loaders: [
+      {
+        test: /\.js$/,
+        loaders: ['babel'],
+        exclude: /node_modules/,
+        include: __dirname
+      }
+    ]
+  }
+}
+
+
+// When inside Redux repo, prefer src to compiled version.
```

Comment:
I recently removed this block from all examples because it was causing some issues. Let’s remove it here as well.